acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/CmdLineParamsConfigFileImpl.java

```diff
@@ -171,6 +173,23 @@ public List<String> createCmdLineParams() {
                 YAML_STRING_FMT,
                 "eth2.import.from",
                 signerConfig.getSlashingImportPath().get().toAbsolutePath()));
+      } else if (signerConfig.getWatermarkRepairParameters().isPresent()) {
```

Comment:
may be we can now refactor a method e.g. `manageSubCommands()` as the list is growing.